src/Chainweb/Pact/Backend/Compaction.hs

```diff
@@ -336,9 +337,11 @@ withTables ts a = do
 -- | Takes a bunch of singleton tablename rows, sorts them, returns them as
 --   @TableName@
 sortedTableNames :: [[SType]] -> [TableName]
-sortedTableNames rows = M.elems $ M.fromListWith const $ flip List.map rows $ \case
-  [SText n@(Utf8 s)] -> (Text.toLower (Text.decodeUtf8 s), TableName n)
-  _ -> error "sortedTableNames: expected text"
+sortedTableNames rows = coerce
```

Comment:
I could have absolutely sworn that I had already fixed this. I am 98% sure this is not fixed as a result of a bad merge or something.

Reply:
To be clear, by "fixed", I only mean that the current code in master has significantly worse readability.